compiler_test.go

```diff
@@ -156,7 +156,7 @@ func testLibX() *Program {
 
 func (r *Runtime) testPrg(p *Program, expectedResult Value, t *testing.T) {
 	vm := r.vm
-	vm.prg = p
+	vm.setPrg(p)
```

Comment:
I created a setter method so that I had a central area that I could add the function tick tracking logic.